Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClientKey.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace BuildXL.Cache.ContentStore.Service.Grpc
+{
+    internal struct GrpcCopyClientKey : IEquatable<GrpcCopyClientKey>
+    {
+        public GrpcCopyClientKey(string host, int grpcPort, bool useCompression)
+        {
+            Host = host;
+            GrpcPort = grpcPort;
+            UseCompression = useCompression;
+        }
+
+        public string Host;

```

Comment:
Could you please move them to the top of the file and make them get-only properties.

And please make the struct readonly.